tools/background_removal/background_removal.xml

```diff
@@ -0,0 +1,64 @@
+<tool id="background_removal" name="Remove image background" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="20.05">
+        <description>with scikit-image</description>  
+    <macros>  
+        <import>creators.xml</import>
+        <import>tests.xml</import>
+        <token name="@TOOL_VERSION@">0.24.0</token>  
+        <token name="@VERSION_SUFFIX@">0</token>  
+    </macros>  
+    <creator>
+        <expand macro="creators/rmassei"/>
+    </creator>
+    <requirements>
+        <requirement type="package" version="0.24.0">scikit-image</requirement>
+    </requirements>
+    <command detect_errors="aggressive">
+        <![CDATA[
+        python "$__tool_directory__/background_removal.py" $input_image $filter $radius $output
+        ]]>
+    </command>
+    <inputs>
+        <param name="input_image" type="data" format="tiff, jpg, jpeg, png, tif" label="Input Image"/>
+        <param name="filter" type="select" label="Select Filter">
+            <option value="rolling_ball">Rolling-Ball Algorithm</option>
+            <option value="dog">Difference of Gaussians</option>
+            <option value="top_hat">Top-Hat Filter</option>
+        </param>
+        <param name="radius" type="float" label="Radius" value="20"/>
+    </inputs>
+    <outputs>
+        <data name="output" format="tiff" label="Background substraction output"/>
+    </outputs>
+    <tests>
+        <test>
```

Comment:
The `img.tif` file is an uint16 image now, right?

I suggest renaming it to `input2_uint16.tif` then. In addition, please also add the uint8 variant back, and name it `input1_uint8.tif`, so we can do the following:

```suggestion
        <!-- uint8 tests -->
        <test>
            <param name="input_image" value="input1_uint8.tif"/>
            <param name="filter" value="dog"/>
            <param name="radius" value="20"/>
            <expand macro="tests/intensity_image_diff" name="output" value="input1_output_dog.tif" ftype="tiff"/>
        </test>
        <!-- uint16 tests -->
        <test>
```